float32 link_quality
float32 signal_strength
